app/models/Review.rb

```diff
@@ -23,4 +24,11 @@ def star_selections
   def self.stars
     ["1 star", "2 stars", "3 stars", "4 stars", "5 stars"]
   end
+
+  def update_overall_rating
+    @sum = (self.food_rating + self.culture_rating + self.nightlife_rating + self.landmark_rating + self.cost_rating + self.weather_rating)*1.0
```

Comment:
Line is too long. [143/80]<br>Redundant `self` detected.<br>Surrounding space missing for operator '*'.